Formula/gq-cli.rb

```diff
@@ -0,0 +1,20 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
```

Comment:
Does "GQ" refer to GoQuant? Also, is go-cli mentioned somewhere in the website? I don't see how the website is related to the software.

```suggestion
  desc "Command-line interface for GoQuant"
```

Reply:
We are a new company and lots of our tools are currently in development. Should we come back later with an updated webpage before submitting another PR?